### DISCUSSION TITLE
Question re: Mass Obsoletion: branch X tickets

### DISCUSSION THREAD

**Discussion by kanems** [monarch-initiative/mondo#6652] [Q&A]
Hi, quick question to make sure I understand the planned work with the mass obsoletion of grouping terms...
I see @sabrinatoro just made a couple of tickets (like https://github.com/monarch-initiative/mondo/issues/6650 and #6551) and there are some specific terms within the ticket.
Are the terms listed explicitly within the ticket the obsoletion candidates, or is the whole branch going to be obsoleted? Thanks for clarifying for me :-)

**Comment by sabrinatoro** [comment]
Hello @kanems 
Thank you for your message. 
No, the whole branch will NOT be obsoleted, only the listed "terms to obsolete" within the branch will be obsoleted. 

We are breaking the big project into smaller pieces by working branch by branch. We will obsolete terms within an ontology branch, and reviewing that the classification of the children that are currently under the terms to be obsoleted. 
This is making our work a little bit easier. 

I hope this makes sense. Sorry if we scarred you :-)
